pages/index.html

```diff
@@ -1,12 +1,196 @@
+        <section class="home" id="home">
+            <aside class="side-bar left-side-bar"></aside>
+            <div class="content">
+                <div class="content-hello">
+                    <h1 id="hello">Hello, World!</h1>
+                </div>
+                <article>
+                    <h2 class="content-item" id="name">I am Lin</h2>
+                    <h2 class="content-item" id="desc">{"A Software Developer in training"}</h2>
+                </article>
+            </div>
+            <aside class="side-bar right-side-bar">
+            </aside>
+            <footer>
+                <img src="../assets/Images/next-page.png" alt="next-page">
+            </footer>
+        </section>
+        <section class="about" id="about">
+            <aside class="side-bar left-side-bar"></aside>
+            <div class="content">
+                <div class="content-item skills">
+                    <h3>SKILLS</h3>
+                    <ul class="my-skills">
+                        <li class="skill ">
+                            <i class="fa-brands fa-python fa-3x"></i>
+                            <p>Python</p>
+                        </li>
+                        <li class="skill">
+                            <i class="fa-solid fa-database fa-3x"></i>
+                            <p>SQL</p>
+                        </li>
+                        <li class="skill">
+                            <i class="fa-brands fa-html5 fa-3x"></i>
+                            <p>HTML</p>
+                        </li>
+                        <li class="skill">
+                            <i class="fa-brands fa-css3 fa-3x"></i>
+                            <p>CSS</p>
+                        </li>
+                        <li class="skill">
+                            <i class="fa-brands fa-js-square fa-3x"></i>
+                            <p>JavaScript</p>
+                        </li>
+                        <li class="skill">
+                            <i class="fa-solid fa-laptop-code fa-3x"></i>
+                            <p>Flask</p>
+                        </li>
+                        <li class="skill">
+                            <i class="fa-brands fa-react fa-3x">
+                            </i>
+                            <p> React</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="content-item" id="my-desc">
+                    <p class="about-desc">
+                        <i class="fa-solid fa-m fa-2x fa-pull-left"></i>
+
+                        y name is Lin, I am a student software developer @ <a href="https://adadevelopersacademy.org/"
+                            id="ada">Ada Developer Academy</a>
+                        Seattle, WA.
+                        <br> I look forward to making a pivot into tech!
+                    </p>
+                    <div class="about-desc">
+                        <P>MY LOCATION</P>
```

Comment:
Style: Since this is a title for this portion of a page it should be a `h3` (since it's on the same level as "SKILLS").